tests/PHPStan/Rules/Methods/data/call-methods.php

```diff
@@ -656,7 +656,7 @@ public function test(callable $str)
 	{
 		$this->test('date');
 		$this->test('nonexistentFunction');
-		$this->test('Test\CheckIsCallable::test');
+		// $this->test('Test\CheckIsCallable::test'); differs between php7/8; tested separately
```

Comment:
moved this line into a separate new test, so I don't need to duplicate the whole `call-methods.php` assertions and can keep the diff small